sentinel/src/schedule/reminders.js

```diff
@@ -110,10 +119,11 @@ module.exports = {
         }
     },
     getClinics: function(options, callback) {
+        const placeTypes = getLeafPlaceTypeIds();
+
         // gets all clinics
-        db.medic.query('medic-client/doc_by_type', {
-            startkey: [ 'clinic' ],
-            endkey: [ 'clinic', {} ],
+        db.medic.query('medic-client/contacts_by_type', {
+            keys: placeTypes,
```

Comment:
I can't add comments that far down, but in `sendReminder` fn, we add the message with the hardcoded recipient `clinic` which might not be resolved for flexible leaf types.

Reply:
I've changed it to "reporting_unit" because we've already found the leaf we want to send it to...